apps/web/lib/features/integrations/calendar/table-time-sheet.tsx

```diff
@@ -381,41 +258,6 @@ export function DataTableTimeSheet({ data, user }: { data?: GroupedTimesheet[],
 				}
 				)}
 			</div>
-			<div className="flex items-center justify-end p-4 space-x-2">
```

Comment:
@Innocent-Akim why that whole block was added and now removed? What was function of it? Did you replaced it with something else or it was not needed at all or what?

Reply:
It was just a function related to table headers. Since the table is no longer used, I thought it was very important to remove it.